user.js

```diff
@@ -270,6 +270,53 @@ user_pref("general.buildID.override",				"20100101");
 // https://github.com/pyllyukko/user.js/issues/120
 user_pref("browser.display.use_document_fonts",			0);
 
+// PREF: Prevent sites from using links like mailto to launch external applications.
+// http://kb.mozillazine.org/Network.protocol-handler.external-default
+// http://kb.mozillazine.org/Network.protocol-handler.warn-external-default
+// https://news.ycombinator.com/item?id=13047883
+// https://bugzilla.mozilla.org/show_bug.cgi?id=167475
+// NOTICE: Breaks mailto links on legit websites, user has to right-click and copy mail address from the link.
```

Comment:
Should be `// NOTICE: Breaks opening third-party mail/messaging/torrent/... clients by clicking on links with custom protocols`